.github/workflows/automation.yml

```diff
@@ -1,8 +1,7 @@
 name: Build & Release WEmulate Frontend
 on:
-  push:
-    tags:
-      - 'v*'
+  release:
+    types: [published]
```

Comment:
You could also use `created`
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release